hnn_core/cell.py

```diff
@@ -137,26 +141,134 @@ def gid(self, gid):
         else:
             raise RuntimeError('Global ID for this cell already assigned!')
 
-    @abstractmethod
-    def get_sections(self):
-        """Get sections in a cell."""
-        pass
-
-    def create_soma(self):
-        """Create soma and set geometry."""
-        # make L_soma and diam_soma elements of self
-        # Used in shape_change() b/c func clobbers self.soma.L, self.soma.diam
-        soma_props = self.soma_props
-
-        self.L = soma_props['L']
-        self.diam = soma_props['diam']
-        self.pos = soma_props['pos']
+    def _set_biophysics(self, p_secs):
+        "Set the biophysics for the default Pyramidal cell."
```

Comment:
docstring for p_secs?

Reply:
Same line: this sets the "biophysics" (membrane channel properties) for _all_ cells, not just pyramidals, no?